help/default/cservice/recover

```diff
@@ -10,8 +10,8 @@ matching you is added (in case of bans Atheme
 can't see), the channel is set invite-only
 and moderated and you are invited.
 
-If you are on channel, you will be opped and
-no ban exception will be added.
+If you are on the channel, you will be opped 
+and a ban exception will be added.
```

Comment:
The grammar for the ban exception could be either way, and the code's not 100% clear to me (specifically what `origin_cu` is)

Can someone confirm https://github.com/atheme/atheme/blob/release/7.2/modules/chanserv/recover.c#L176?

Seems like pretty weird logic in general to me, IMO